config/core/configmaps/autoscaler.yaml

```
@@ -182,3 +182,9 @@ data:
     # allow-zero-initial-scale controls whether either the cluster-wide initial-scale flag,
     # or the "autoscaling.knative.dev/initialScale" annotation, can be set to 0.
     allow-zero-initial-scale: "false"
+
+    # default-max-scale is the cluster-wide default value for the max
+    # scale of a revision after creation, unless overridden by the
```

the "after creation" language is specific to "initScale" (which only applies just after creation)

```suggestion
    # scale of a revision, unless overridden by the
```